domains/swarim.JSON

```
@@ -0,0 +1,12 @@
+{
+  "description": "Swarim - Software Developer Portfolio",
+  "domain": "github.io", 
+  "subdomain": "swarim",
+  "owner": {
+    "repo": "https://github.com/swarimabdussamad/swarim.is-a-dev",
```

add username and fix repo
```suggestion
  "repo": "https://github.com/swarimabdussamad/swarim.is-a-dev",
  "owner": {
    "username": "swarimabdussamad",
```